_projects/2023/100061715/100061715.Rmd

```diff
@@ -0,0 +1,256 @@
+---
+title: "Unveiling birth clinical practice in Spain: mapping the cesarian sections and birth dynamics"
+description: "Exploring the nuances of maternal healthcare in Spain through an augmented map, showcasing original cesarean section rates alongside additional insights such as total births per autonomous community and cesarean rates adjusted for the average maternal age."
+categories: 2023
+author: "Celia Muñoz"
+date: "`r Sys.Date()`"
+---
```

Comment:
Fixes required:

- 2023 should be quoted.
- The output should be a distill document. See e.g. [this](https://github.com/IBiDat/dataviz/blob/main/_projects/2022/100348375/100348375.Rmd#L1-L16) for an example of the yaml header.